common/src/main/java/com/wynntils/gui/screens/maps/GuildMapScreen.java

```diff
@@ -309,6 +352,41 @@ private void renderHoveredTerritoryInfo(PoseStack poseStack) {
     private void renderPois(PoseStack poseStack, int mouseX, int mouseY) {
         List<Poi> pois = Managers.Territory.getTerritoryPoisFromAdvancement();
```

Comment:
You should send in territoryDefenseFilterType to getTerritoryPoisFromAdvancement, and let the filtering happen there. Or maybe create a new `getFilteredTerritoryPois` in the  TerritoryManager.